requirements.txt

```diff
@@ -1,4 +1,5 @@
-Django==3.2.2
+#Django==3.2.2
+Django~=3.1.7
```

Comment:
We should keep 3.2.2 - I spent too much time allowing code for 3.2.x!

Reply:
this creating dependency error while installing requirements.xtx as other apps like dbcopy is installed with django 3.1.7